test/metabase/models/permissions_group_test.clj

```diff
@@ -131,3 +133,17 @@
           ;; Only check this for dbs with permissions
           (when (contains? dbs-in-perms db-id)
             (is (= #{db-id} (->> graph :groups vals (mapcat keys) set)))))))))
+
+(deftest set-default-permission-values!-test
+  (testing "A new group has no permissions for any DBs by default"
+    (mt/with-temp [:model/Database         {db-id :id}    {}
+                   :model/PermissionsGroup {group-id :id} {}]
+      (is
+       (= {group-id
+           {db-id
+            {:data-access :no-self-service,
+             :download-results :no,
+             :manage-table-metadata :no,
+             :native-query-editing :no,
+             :manage-database :no}}}
```

Comment:
is this a magic map? Should we generate it from Permissions data structure?

Reply:
This was already in the feature branch